test/fuzz/acme/roundtrip_test.go

```diff
@@ -22,8 +22,9 @@ import (
 	"k8s.io/apimachinery/pkg/api/apitesting/roundtrip"
 
 	acmefuzzer "github.com/jetstack/cert-manager/internal/apis/acme/fuzzer"
+	acmeinstall "github.com/jetstack/cert-manager/internal/apis/acme/install"
 )
 
 func TestRoundTripTypes(t *testing.T) {
-	roundtrip.RoundTripTestForAPIGroup(t, Install, acmefuzzer.Funcs)
+	roundtrip.RoundTripTestForAPIGroup(t, acmeinstall.Install, acmefuzzer.Funcs)
 }
```

Comment:
Sorry, I didn't pay close enough attention in my last review.
This test is hermetic and should remain in its original location.

Reply:
reasonable take yeah; I don't like having both `internal/test` and `test/internal` but sure, I'll move these back and aim to make the minimal possible change to allow `go test ./internal/...`